packages/vue/src/components/FCheckboxField/FCheckboxField.vue

```diff
@@ -89,9 +90,10 @@ export default defineComponent({
     },
     emits: ["change", "update:modelValue"],
     setup() {
+        const { showDetails, getFieldsetLabelText } = useFieldset();
```

Comment:
Switching to using the `useFieldset()` composable as I don't think it is a very good design to manually inject these. It makes refactoring more difficult.

The same change is present for `FRadioField`.